### PR TITLE
Mfranz1/finish reset password implementation

### PR DESCRIPTION
**Pull request by mfranz1**

### CODE REVIEW

**Review by Maliyaa — APPROVED** [review]

**Review by bereketsega — APPROVED** [review]

File: src/app/shared/pages/help-page/copy.ts [hunk 1]
```diff
@@ -77,22 +77,41 @@ export const sections = {
...
+                            `</p><img 
+                                src="../../../assets/conditionbuildertech.png"
+                                style="width: 100%; max-width: 600px;"
+                            ></img><br><p>` +
+                            `<b>Technology</b> is the technology an actor may use for this competency (i.e. Windows 10, Linux, etc.). ` +
+                            `To add technology to your competency, press either the <b>TAB</b> or <b>ENTER</b> key.<br>` +
+                            // `</p><img
+                            //     src="../../../assets/conditionbuildertech.png"
+                            //     style="width: 100%; max-width: 600px;"
+                            // ></img><br><p>` +
```
Comment (pzalep1) [inline src/app/shared/pages/help-page/copy.ts:100]: Why is this commented out?

**Review by pzalep1 — COMMENTED** [review]
I think you might have some stuff from your other branches in here @mfranz1

File: src/app/shared/pages/help-page/copy.ts (hunk 1 above)
Comment (mfranz1) [reply src/app/shared/pages/help-page/copy.ts:100]: Placeholder. Fileupload was broken I couldn't take a screen grab earlier

**Comment by JosueMolinaMorales** [comment]
Can you add screenshots

**Review by Cwagne17 — APPROVED** [review]

**Pull request merged**